docs/release-notes/2.4.1.md

```diff
@@ -0,0 +1,3 @@
+## Changes:
```

Comment:
Also mention fixing the bug in big number chai matchers:
previously `to.be.gt` didn't work
also `expect(3).to.be.at.least(utils.bigNumberify(2))` didn't pass

Reply:
And this applies to all operations that take the order of arguments into account, so `>` `>=` `<` and `<=`